src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -780,6 +784,13 @@ public void zremrangeByRank(final byte[] key, final long start, final long end)
     sendCommand(ZREMRANGEBYRANK, key, toByteArray(start), toByteArray(end));
   }
 
+  public void zremrangeByScore(final byte[] key, final double start, final double end) {
+    byte[] byteArrayStart = (start == Double.NEGATIVE_INFINITY) ? "-inf".getBytes() : toByteArray(start);
```

Comment:
I think this kind of conversion is bad cause giving -inf -inf as score range is still valid. It should check both Double.NEGATIVE_INFINITY and Double.POSITIVE_INFINITY.